sorting_test.go

```diff
@@ -71,33 +71,50 @@ func readLines(path string) ([]string, error) {
 	var lines []string
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
+		// ignore blank lines
+		d := scanner.Text()
```

Comment:
This extra check is not needed. The code should assume valid input to focus on algorithm and reduce overhead code.